CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
The code of conduct shouldn't be on the docs repo?

Reply:
Do you think so? I was thinking that it's useful to each repo has its CODE_OF_CONDUCT.md, but if you think that it make sense to  move in one page for all repos, I will do :smiley:

Reply:
I guess it's not that is going to be modified, so it's ok like this.